seatunnel-engine/seatunnel-engine-common/src/main/java/org/apache/seatunnel/engine/common/config/server/ServerConfigOptions.java

```diff
@@ -49,6 +49,8 @@ public class ServerConfigOptions {
 
     public static final Option<Integer> CHECKPOINT_STORAGE_MAX_RETAINED = Options.key("max-retained").intType().defaultValue(1).withDescription("The maximum number of retained checkpoints.");
 
+    public static final Option<String> QUEUE_TYPE = Options.key("queue_type").stringType().defaultValue("disruptor").withDescription("The internal data cache queue type.");
```

Comment:
The value should be enum, also please use `blockingQueue` at default value. Then change doc